subtitles.md

```diff
@@ -472,4 +472,54 @@ This Block does not need a BlockAddition as the Cue did not contain an Identifie
 
 ## Storage of WebVTT in Matroska vs. WebM
 
-Note: the storage of WebVTT in Matroska is not the same as the design document for storage of WebVTT in WebM. There are several reasons for this including but not limited to: the WebM document is old (from February 2012) and was based on an earlier draft of WebVTT and ignores several parts that were added to WebVTT later; WebM does still [not support subtitles at all](http://www.webmproject.org/docs/container/); the proposal suggests splitting the information across multiple tracks making demuxer's and remuxer's life very difficult.
\ No newline at end of file
+Note: the storage of WebVTT in Matroska is not the same as the design document for storage of WebVTT in WebM. There are several reasons for this including but not limited to: the WebM document is old (from February 2012) and was based on an earlier draft of WebVTT and ignores several parts that were added to WebVTT later; WebM does still [not support subtitles at all](http://www.webmproject.org/docs/container/); the proposal suggests splitting the information across multiple tracks making demuxer's and remuxer's life very difficult.
+
+
+# HDMV presentation graphics subtitles
+
+The specifications for the HDMV presentation graphics subtitle format (short: HDMV PGS) can be found in the document "Blu-ray Disc Read-Only Format; Part 3 — Audio Visual Basic Specifications" in section 9.14 "HDMV graphics streams".
+
+## Storage of HDMV presentation graphics subtitles
+
+### CodecID & CodecPrivate: codec identification
+
+The CodecID to use is <tt>S_HDMV/PGS</tt>. A CodecPrivate element is not used.
```

Comment:
I have to admit that I've only copied that section from the WebVTT section where `<tt>` was used. But I agree, backticks should suffice. In a further PR we can remove the use of `<tt>` from other places as well.